apps/wallet/src/ui/app/staking/stake/StakeForm.tsx

```diff
@@ -141,7 +141,10 @@ function StakeForm({
                                 weight="medium"
                                 color="steel-darker"
                             >
-                                {unstake ? 0 : calculateRemaining} {symbol}
+                                {calculateRemaining < 0
```

Comment:
Why can this happen?

Reply:
Yes, if `amount` entered is  bigger than the available SUI you own. `maxToken - amount`